docs/manage-relational-metadata-using-gravitino.md

```diff
@@ -23,6 +23,7 @@ For more details, please refer to the related doc.
 - [**PostgreSQL**](./jdbc-postgresql-catalog.md)
 - [**Doris**](./jdbc-doris-catalog.md)
 - [**Apache Iceberg**](./lakehouse-iceberg-catalog.md)
+- [**Apache Paimon**](./lakehouse-paimon-catalog.md)
```

Comment:
There're many places need to add Paimon, not just only here.

Reply:
OK, let me check it thoroughly.

Reply:
You can also let @FANNG1 or @caican00 to modify the doc if you're not sure about Paimon part.

Reply:
@caican00 is planing to propose a PR to fix the document today.

Reply:
Do you mean @caican00 will modify the Paimon-related content in the file `manage-relational-metadata-using-gravitino.md`? If so, should I make any changes to this PR?

Reply:
If @caican00 will work on this, @yuqi you can leave this file as it is.

Reply:
OK, I will revert changes to this file, if @caican00 has no time to do it, I would raise a another PR to fix it.

Reply:
done